lib/smb2/packet/create_request.rb

```diff
@@ -40,5 +44,9 @@ class CreateRequest < Smb2::Packet
 
     rest :buffer
 
+    # @return [Symbol] a key in {Smb2::COMMANDS}
+    def self.command
```

Comment:
Unnecessary if you move to [class-constant lookup](https://github.com/rapid7/smb2/pull/28/files#r28820364)